crossdock/docker-compose.yml

```diff
@@ -3,6 +3,7 @@ version: '2'
 services:
     crossdock:
         image: crossdock/crossdock
+        dns_search: .  # Ensures unified DNS config.
```

Comment:
Could you elaborate on how this ensures unified DNS config?
Isn't  `.` an invalid IP Address or host name?

Reply:
I wish I understood this. I can try to get some more info and will post here.

Reply:
OK have a better solution and explained in comment.

Reply:
What is the problem being solved?

Reply:
Well using `8.8.8.8` showed me that using a real DNS search host can mess up the test. The docker-compose hostnames won't be resolved correctly. The motive for using `dns_search: .` is described [here](https://github.com/crossdock/crossdock/pull/64). I will add a link in the comment.